### PR TITLE
Scroll animate

### PR DESCRIPTION
**Pull request by AcquahLopid**
added just basic styles to the front page, Testing!!!!!

### CODE REVIEW

File: assets/script.js [hunk 1]
```diff
@@ -272,6 +272,7 @@ $(document).ready(function() {
         {
             $("#chooseKeyWord-btn").css("display", "none");
             pickMovie(genre, keywords);
+            console.log(pickMovie(genre, keywords))
```
Comment (LiubovSobolevskaya) [inline assets/script.js:275]: It's just for debugging, remove before you pull request

File: index.html [hunk 2]
```diff
@@ -8,15 +8,20 @@
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css"
     />
+    <!-- <link
```
Comment (LiubovSobolevskaya) [inline index.html:11]: It's a comment! Remove before pull reqiest!

File: index.html [hunk 3]
```diff
@@ -52,6 +57,6 @@
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
     <script src="./assets/movie-script.js"></script>
     <script src="./assets/script.js"></script>
-    
+    <!-- <script src="https://cdn.jsdelivr.net/npm/aos@3.0.0/dist/aos.js"></script> -->
```
Comment (LiubovSobolevskaya) [inline index.html:60]: remove!

File: index_movies.html [hunk 4]
```diff
@@ -110,4 +110,5 @@
 
 });
 </script>
-</html>
\ No newline at end of file
+<script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
```
Comment (LiubovSobolevskaya) [inline index_movies.html:113]: what does this library do?

**Review by LiubovSobolevskaya — CHANGES_REQUESTED** [review]

**Review by Sabeen44 — APPROVED** [review]

**Pull request closed without merging**